spinetoolbox/spine_db_editor/widgets/graph_view_mixin.py

```diff
@@ -165,7 +165,8 @@ def _accepts_entity_item(self, item, db_map):
             if not self._parameter_value_accepts(item, db_map):
                 return False
         if not self._scenario_accepts(item, db_map):
-            return False
+            if not self._parameter_value_accepts(item, db_map):
```

Comment:
This could be combined with the `if` statement above with `and` operator:

```python
if not self._scenario_accepts(item, db_map) and not self._parameter_value_accepts(item, db_map):
```